.gitignore

```diff
@@ -1,4 +1,5 @@
 build
 composer.lock
 docs
-vendor
\ No newline at end of file
+vendor
+.phpunit.result.cache
```

Comment:
The cached PHPUnit file is generated by `PHPUnit 8.x` version.

And it should let above file ignore Git version control.